Cargo.toml

```diff
@@ -23,6 +23,10 @@ semver = "0.5.0"
 separator = "0.3.1"
 time = "0.1.35"
 
+[dependencies.clippy]
+optional = true
+version = "0.0.118"
```

Comment:
I'm going to set this to update automatically and see how much trouble it gives us. At any rate, it'll give us a better idea of what types of changes come in clippy updates.